worldedit_commands/init.lua

```diff
@@ -1165,7 +1165,11 @@ minetest.register_chatcommand("/allocate", {
 			worldedit.player_notify(name, "File was created with newer version of WorldEdit!")
 		end
 		local nodepos1, nodepos2, count = worldedit.allocate(pos, value)
-
+		if not nodepos1 or not nodepos2 or not count then
+			worldedit.player_notify(name, "0 nodes allocated")

```

Comment:
I did __not__ allocate a region in this case, this is misleading.

Reply:
This fixes the issue(crash at //allocate). So you mean, that the notification message is wrong? Can you merge the pull request and change the notification message? Or what do you mean? Changing the message shouldn't be a problem, I think. When I should change the message before merging, what should the message be? Any suggestions?